src/lib/get.ts

```diff
@@ -0,0 +1,426 @@
+        const value = map[key];
+        if (value === undefined) {
+            throw new ConvertException(`Map: A value for key '${key}' does not exist.`);
+        } else {
+            return value;
+        }
+    },
+
+    // Point, Frame, Beat, Init code fields - reorder to I,F,B,P order.
+    CodePFBI(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 3],
+            ['perFrame', 1],
+            ['onBeat', 2],
+            ['perPoint', 0]
+        ];
+        return this.CodeSection(blob, offset, map);
+    },
+
+    // Frame, Beat, Init code fields - reorder to I,F,B order.
+    CodeFBI(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 2],
+            ['perFrame', 1],
+            ['onBeat', 0]
+        ];
+        return this.CodeSection(blob, offset, map);
+    },
+
+    CodeIFBP(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 0],
+            ['perFrame', 1],
+            ['onBeat', 2],
+            ['perPoint', 3]
+        ];
+        return this.CodeSection(blob, offset, map);
+    },
+
+    CodeIFB(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 0],
+            ['perFrame', 1],
+            ['onBeat', 2]
+        ];
+        return this.CodeSection(blob, offset, map);
+    },
+
+    // used by 2.8+ 'Effect List'
+    CodeEIF(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 0],
+            ['perFrame', 1]
+        ];
+        const code: [CodeSection, number] = this.CodeSection(blob, offset, map);
+        return [
+            {
+                enabled: this.Bool(blob, offset, config.sizeInt)[0],
+                init: code[0]['init'],
+                perFrame: code[0]['perFrame']
+            },
+            code[1]
+        ];
+    },
+
+    // used only by 'Global Variables'
+    NtCodeIFB(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 0],
+            ['perFrame', 1],
+            ['onBeat', 2]
+        ];
+        return this.CodeSection(blob, offset, map, /*nullterminated*/ true);
+    },
+
+    // used only by 'Triangle'
+    NtCodeIFBP(blob: Uint8Array, offset: number): [CodeSection, number] {
+        const map: [string, number][] = [
+            ['init', 0],
+            ['perFrame', 1],
+            ['onBeat', 2],
+            ['perPoint', 3]
+        ];
+        return this.CodeSection(blob, offset, map, /*nullterminated*/ true);
+    },
+
+    // the 256*-functions are used by ancient versions of 'Super Scope', 'Dynamic Movement', 'Dynamic Distance Modifier', 'Dynamic Shift'
+    LegacyCodePFBI(blob: Uint8Array, offset: number): [CodeSection, number] {
```

Comment:
I'm okay with (though not a big fan) of changing the names of the functions to `Legacy*`, but then the comment has to reflect this change as well.

Reply:
Ah, I suppose the change was because the function name would've started with `256` now. Makes sense.